src/components/note/Note.svelte

```diff
@@ -77,13 +77,19 @@
 					<time datetime={note.data.timestamp.toISOString()} class="font-mono text-2.6 c-remark">{Time(note.data.timestamp)}</time>
 				</div>
 				<span class="flex items-center gap-1 sm:ml-a c-remark">
-					{#each note.data.tags as tag}
+					{#each note.data.tags || [] as tag}
 						<button onclick={() => switch_tag(tag, true)} class="text-3.5 sm:text-sm">#{tag}</button>
 					{/each}
 				</span>
 			</section>
 		{/each}
 
+		{#if notes.length === 0 || filtered.length === 0}
```

Comment:
That’s logically sufficient
```suggestion
		{#if filtered.length === 0}
```